common/model_package.go

```diff
@@ -187,6 +187,25 @@ func FuzzySearchPackages(
 	return scanPackageModels(session.Query(query))
 }
 
+func InsertPackage(
```

Comment:
@skeswa Is this a func declaration style we want to be consistent with in other files?